ale_linters/go/gometalinter.vim

```diff
@@ -1,8 +1,9 @@
-" Author: Ben Reedy <https://github.com/breed808>
+" Author: Ben Reedy <https://github.com/breed808>, Jeff Willette <jrwillette88@gmail.com>
 " Description: Adds support for the gometalinter suite for Go files
 
 call ale#Set('go_gometalinter_options', '')
 call ale#Set('go_gometalinter_executable', 'gometalinter')
+call ale#Set('go_gometalinter_lint_package', 0)
```

Comment:
Maybe call this `go_metalinter_package` to make it just a bit less stuttery?

Reply:
I disagree, the precedent that has been set is clearly `[lang]_[tool]_[option]`. There is no tool called `metalinter` and it would be very confusing seeing the pattern of the other options broken with this rename

Reply:
Uuh... `gometalinter` is definately a [tool](https://github.com/alecthomas/gometalinter), so that should indeed be `[lang] => go`, `[tool] => gometalinter` and `[option] => package` Or am I missing anything?

Seems to be perfectly inline with all the other namings in this file:

`go_gometalinter_options`
`go_gometalinter_executable`

As a matter of fact, following your own reasoning it seems you have added a new pattern yourself `[lang]_[tool]_[option]_[???]`.

But hey, naming is hard right? :wink: It just caught my eye, so I thought to make a suggestion.

Reply:
> Maybe call this go_metalinter_package to make it just a bit less stuttery?

according to your response here the lang is `go` and the tool is `metalinter` which does not exist and could be a name of another tool. `gometalinter` is the name of the tool. IMO the option requires two words in order to be clear so `lint_package` is the option

Reply:
Oh, I made a type there... I intended to write `go_gometalinter_package` 😏